src/heretic/g_game.c

```diff
@@ -1990,6 +1990,21 @@ void G_DoPlayDemo(void)
     for (i = 0; i < MAXPLAYERS; i++)
         playeringame[i] = (*demo_p++) != 0;
 
+    {
```

Comment:
How about putting these into a static helper function? I mean, "don't repeat yourself" and stuff. 😉

Reply:
Sure, we can extract it. The playdemo and timedemo methods themselves are mostly duplicated, which is why I didn't bother 😅

Reply:
Are the any known cases where `consoleplayer != 0`?

Reply:
It's for coop demos. This will let you play back a demo with player 2 as the consoleplayer for example.

Reply:
I get this. What I mean is, are there demos out there in the wild that require this to keep sync?

Reply:
Ah ok. No, this doesn't affect demo sync. It's purely for viewing purposes.